src/app/frontend/cluster/cluster.html

```diff
@@ -14,6 +14,19 @@
 limitations under the License.
 -->
 
+<div layout="row">
+  <kd-graph-card graph-title="[[CPU usage|Title for graph card displaying CPU metric of one all resources.]]"
+                 metrics="::$ctrl.cluster.nodeList.cumulativeMetrics"
```

Comment:
Ye. I will outline all changes once everything is finished. Usually if I come across small issue/missing code I'm trying to add/fix it. If it's something bigger then I am creating new PR.